research/src/pages/components/popover.research.explainer.md

```diff
@@ -300,61 +300,56 @@ Be mindful when using other stylesheets that you haven't authored. These may set
 
 ### Animation of Popovers
 
-The show and hide behavior for popovers is designed to make animation of the show/hide trivially easy:
+An early version of the Popover API included custom behaviors to make animations and transitions of popovers trivially easy. During the standardization process, however, this functionality was removed, with the intention to replace it with a more general capability for the web platform that also applies to other top layer elements. This section describes how that new set of capabilities will work together to help animate popovers (and modal `<dialog>`s).
+
+The new capabilities include the ones described in these five CSSWG issues:
+
+- https://github.com/w3c/csswg-drafts/issues/4441 (animation of discrete CSS properties)
+- https://github.com/w3c/csswg-drafts/issues/6429 (animating `display`)
+- https://github.com/w3c/csswg-drafts/issues/8189 (animating to and from the top layer)
+- https://github.com/w3c/csswg-drafts/issues/8174 (the "initial" frame issue)
+- https://github.com/w3c/csswg-drafts/issues/8389 (`inert`ing things that animate to `display:none`)
+
+Assuming all five of the above features land in specs and browsers in roughly the form they're currently being discussed, then to animate a popover's show/hide transitions you would use:
 
 ```css
 [popover] {
-  opacity: 0;
-  transition: opacity 0.5s;
+  --transition-time: 0.5s;
+  transition: display var(--transition-time), overlay-behavior var(--transition-time), opacity var(--transition-time);
```

Comment:
@flackr would you mind reviewing my examples and descriptions in this section, to make sure I'm not lying?